docs/releases/Release-process.md

```diff
@@ -1,81 +0,0 @@
-# Release Process
-
-This page documents the processes involved with creating new Flutter releases. If you're interested in learning more about why certain branch points are chosen, or how releases are built, continue reading. If you're looking for the current release info, see the appendix at the bottom.
```

Comment:
I think some of this document is actually informational, and can actually stay here. I think I likely need to take a pass through this, and edit out all of the process specific parts, as those will change over time, and we shouldn't duplicate the details if we don't sync them.